### PR TITLE
MLCronJob status update

### PR DESCRIPTION
**Pull request by jwierzbo**
https://github.com/arangodb/kube-arangodb-enterprise/pull/83

### CODE REVIEW

**Review by nikita-vanyasin — APPROVED** [review]

File: chart/kube-arangodb/templates/ml-operator/role.yaml [hunk 1]
```diff
@@ -52,5 +52,9 @@ rules:
         - "secrets"
         - "serviceaccounts"
       verbs: ["*"]
+    - apiGroups: ["batch"]
```
Comment (ajanikow) [inline chart/kube-arangodb/templates/ml-operator/role.yaml:55]: Check few lines above

File: examples/ml/ml-cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+apiVersion: ml.arangodb.com/v1alpha1
```
Comment (ajanikow) [inline examples/ml/ml-cronjob.yaml:1]: Remove example, we will provide example in docs later

File: pkg/apis/ml/v1alpha1/extension_conditions.go [hunk 3]
```diff
@@ -29,4 +29,7 @@ const (
 	ExtensionMetadataServiceValidCondition api.ConditionType = "MetadataServiceValid"
 	ExtensionServiceAccountReadyCondition  api.ConditionType = "ServiceAccountReady"
 	LicenseValidCondition                  api.ConditionType = "LicenseValid"
+	CronJobCreatedCondition                api.ConditionType = "CronJobCreated"
```
Comment (ajanikow) [inline pkg/apis/ml/v1alpha1/extension_conditions.go:32]: Remove conditions

**Review by ajanikow — CHANGES_REQUESTED** [review]

File: examples/ml/ml-cronjob.yaml (hunk 2 above)
Comment (jwierzbo) [reply examples/ml/ml-cronjob.yaml:1]: why? it is useful for quick tests...

Comment (ajanikow) [reply examples/ml/ml-cronjob.yaml:1]: We will generate it in examples and readme for all objects

**Review by ajanikow — APPROVED** [review]

**Pull request merged**